pkg/aspect/manager.go

```diff
@@ -61,3 +66,12 @@ type (
 		Execute(attrs attribute.Bag, mapper expr.Evaluator) (*Output, error)
 	}
 )
+
+func structToProto(in *structpb.Struct, out proto.Message) error {
```

Comment:
This func feels a bit out of place here (especially since jsonpb, structpb, etc weren't included previously), but shoving it off to a util.go file in this package doesn't feel great either.